src/azure-cli-core/setup.py

```diff
@@ -69,7 +69,7 @@
     'requests~=2.20',
     'six~=1.12',
     'wheel==0.30.0',
-    'azure-mgmt-resource~=6.0',
+    'azure-mgmt-resource~=8.0.1',
```

Comment:
This will lock the version to be 8.0.*. Is this wanted behavior?

Reply:
I understand. We'd better lock it in case we hit issues like https://github.com/Azure/azure-cli/pull/10407 again.